modules/system-info/reporters/server.php

```diff
@@ -126,6 +128,45 @@ public function get_php_version() {
 		return $result;
 	}
 
+	/**
+	 * Get PHP memory limit.
+	 *
+	 * Retrieve the PHP memory limit.
+	 *
+	 * @return array {
+	 *    Report data.
+	 *
+	 *    @type string $value          PHP memory limit.
+	 *    @type string $recommendation Recommendation memory limit.
+	 *    @type bool   $warning        Whether to display a warning. True if the limit
+	 *                                 is below the recommended 64M, False otherwise.
+	 * }
+	 */
+	public function get_php_memory_limit() {
+		$result = [
+			'value' => (string) WP_Site_Health::get_instance()->php_memory_limit,
+		];
+
+		$min_recommended_memory = '64M';
```

Comment:
```suggestion
	 *                                 is below the recommended 512M, False otherwise.
	 * }
	 */
	public function get_php_memory_limit() {
		$result = [
			'value' => (string) WP_Site_Health::get_instance()->php_memory_limit,
		];

		$min_recommended_memory = '512M';
```

@matipojo

Reply:
"WP Memory limit of 128 MB or greater (256 MB or higher is preferred)"

https://elementor.com/help/requirements/

Reply:
Handled.